custom/zutil.cli.doc_import.php

```diff
@@ -0,0 +1,165 @@
+<?php
+/**
+ * Command-line / Unattended document import utility.
+ *
+ * Expected arguments:
+ * site - As required for all cronjobs
+ * path - $GLOBALS key specifying the path
+ * pid - patient id to be used for all selected documents
+ * category - encoded category description to assign all selected documents
+ * limit - Maximum number of files to be imported
+ * in_situ - Retain documents in current folder
+ *
+ * Examples:
+ * 1. To import received faxes:
+ *    a. Set 'Scanner Directory' to network location where a fax machine can save received files
+ *    b. Schedule cronjob '/usr/bin/php -f /var/www/html/emr/custom/zutil.cli.doc_import.php site=default'
+ *    c. View and process received faxes using 'New Documents' menu.
+ *
+ * 2. To import collection of medical record files for a specific patient:
+ *    a. Save files in Scanner Directory
+ *    b. For patient nnn and category 'Patient Records', access this script online with request parameters as:
+ *       zutil.cli.doc_import?site=default&pid=nnn&category=Patient+Records&limit=1000
```

Comment:
This use of thi script won't be allowed though, correct? (since I assumed the allow_cronjobs.php script only allowed it to be run as a cli or at least that was my impression, which is needed to ensure security of these scripts):
https://github.com/openemr/openemr/blob/master/library/allow_cronjobs.php#L13

Reply:
Line 13 block in `allow_cronjobs` returns the control to parent without affecting $ignore_auth.  Parent script then invokes globals.php that takes care of online access/security.

We would use this in online dropdowns where the request is built with current $pid.

Reply:
thanks for the explanation. will bring in this script into the codebase. thanks for the contribution!